airbyte-integrations/connectors/destination-mongodb/build.gradle

```diff
@@ -21,4 +21,6 @@ application {
 
 dependencies {
     implementation 'commons-codec:commons-codec:1.16.0'
+
+    testImplementation 'org.testcontainers:mongodb:1.19.0'
```

Comment:
we're no longer importing the cdk from source, so the dependencies don't apply transitively any more.